.gitignore

```diff
@@ -1,3 +1,5 @@
 coverage/
 node_modules/
 test/output/
+.nyc_output/
```

Comment:
This is an acceptable change, but could you do it in a separate PR?
I prefer to have each PR be self-contained

Reply:
Done.
Maybe we should squash this when merging. Didn't have the time to do that.